test/cpp/interop/server_helper.h

```diff
@@ -43,6 +44,16 @@ namespace testing {
 
 std::shared_ptr<ServerCredentials> CreateInteropServerCredentials();
 
+class InteropContextInspector {
+ public:
+  InteropContextInspector (const ::grpc::ServerContext& context);
```

Comment:
Actually, lemme introduce a placeholder method to be removed once an actually useful one takes its place to appease the gods of msan.